src/views/settings/SettingsExternalServices.tsx

```diff
@@ -52,33 +54,46 @@ const SettingsExternalServices: Screen<"SettingsExternalServices"> = ({
     info += `Établissement : ${account.authentication.schoolID || "N. not"}\n`;
 
 
-    Alert.alert(
-      "Informations du compte",
-      info,
-      [
-        { text: "OK", style: "cancel" },
+    showAlert({
+      title: "Informations du compte",
+      message: info,
+      icon: <BadgeInfo />,
+      actions: [
         {
-          text: "Supprimer",
-          style: "destructive",
-          onPress: () => confirmDeleteAccount(account)
-        }
+          title: "OK",
+          icon: <Check />,
+          primary: true,
+        },
+        {
+          title: "Supprimer",
+          icon: <Trash2 />,
+          onPress: () => confirmDeleteAccount(account),
+          danger: true,
+        },
       ]
-    );
+    });
   };
 
   const confirmDeleteAccount = (account: any) => {
-    Alert.alert(
-      "Supprimer le compte",
-      "Es-tu sûr de vouloir supprimer ce compte ?",
-      [
-        { text: "Annuler", style: "cancel" },
+    showAlert({
+      title: "Supprimer le compte",
+      message: "Es-tu sûr de vouloir supprimer ce compte ?",
+      icon: <BadgeHelp />,
+      actions: [
+        {
+          title: "Annuler",
+          icon: <Undo2 />,
+          primary: true,
+        },
         {
-          text: "Supprimer",
-          style: "destructive",
-          onPress: () => removeAccount(account.localID)
+          title: "Supprimer",
```

Comment:
```suggestion
          title: "Confirmer",
```
Pour être en accord avec les autres alerts